docs/UserGuide.md

```diff
@@ -141,6 +141,70 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Listing all members for attendance : `attendance`
+
+Lists all contacts that have the role “Member” in the address book, making it easier to check and mark attendance for them directly.
+
+Format: `attendance` or `atd` (alias)
+
+* The command must be in lowercase. Variations (e.g., capitalized or mixed-case) will not be recognized.
+* This command does not take any additional parameters. If any extra input is provided, an error message will be displayed.
+
+
+![result for 'attendance'](images/ListAttendanceResult.png)
+
+### Marking attendance : `mark`
+
+Marks attendance for **members** with specified Telegram handles on a specific date.
+
+
+Format: `mark t/TELEGRAM1 [t/TELEGRAM2] [...] d/DATE`
+
+Alternative Format: `m t/TELEGRAM1 [t/TELEGRAM2] [...] d/DATE`
+
+* Each Telegram handle must begin with t/, followed by an alphanumeric or _ string with no spaces, e.g., t/berniceyu123.
+* The command is not case-sensitive (e.g., t/alexyeoh is the same as t/ALEXYEOH) except command keyword `mark` itself.
+* The command does not allow spaces between the prefix t/ or d/ and the argument.
+
+Command Parameters:
+- Telegram Handles (`t/telegram`):
+  * Accepts multiple handles separated by spaces, each beginning with t/.
+  * Handles must be alphanumeric or include _ (e.g., t/alex_yeoh123).
+  * Invalid input: Spaces within or after the prefix (e.g., t/ alexyeoh) result in an error.
+  * Telegram handle should belong to a contact with `Member` role.
+
+- Date (`d/date`)
+  * Takes in one string value with format d/YYYY-MM-DD (eg. d/2024-01-22 not d/2024/1/22)
+  * Only accept one date, if multiple dates are input, only the last one will be recorded as the attendance date.
+  * Date must not be later than current date (eg. 2090-11-02 is an invalid datetime)
+
+Examples:
+
+* `mark t/toom t/maary d/2024-11-02`
```

Comment:
is 'toom' and 'maary' intentional or typo haha

Reply:
it's the telegram handle of the contacts, setting to distinguish from the contacts' actual name

Reply:
oh ok